jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml

```diff
@@ -61,7 +61,7 @@
     - kubernetes-soak-gce-federation-deploy:
         blocker: ci-kubernetes-soak-gce-federation-test
         job-name: ci-kubernetes-soak-gce-federation-deploy
-        frequency: 'H 0 * * 2'
+        frequency: 'H 0 * * *'
```

Comment:
Ok makes sense.

Can you please open an issue to track that this must be reverted?